### PR TITLE
add vdiffr tests back in

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: tests/testthat/test-vdiffr_geoms_coords.R [hunk 1]
```diff
@@ -1,6 +1,4 @@
-.runThisTest <- Sys.getenv("RunAllseeTests") == "yes"
-
-if (.runThisTest && getRversion() >= "4.1" && require("vdiffr") && require("ggplot2")) {
+if (getRversion() >= "4.1" && require("vdiffr") && require("ggplot2")) {
```
Comment (strengejacke) [inline tests/testthat/test-vdiffr_geoms_coords.R:1]: make sure all tests are skipped on CRAN, e.g. by adding `.runThisTest` to the if-statement.

**Review by strengejacke — CHANGES_REQUESTED** [review]

File: tests/testthat/test-vdiffr_geoms_coords.R (hunk 1 above)
Comment (IndrajeetPatil) [reply tests/testthat/test-vdiffr_geoms_coords.R:1]: But they are also skipped locally on my computer, which is suboptimal. I have added `skip_on_cran()` to all these tests, and they are not run on CRAN by default anyway. So we can't be more careful here.

File: R/geom_from_list.R [hunk 2]
```diff
@@ -111,20 +111,21 @@
...
+#' if (require("tidygraph", quietly = TRUE)) {
+#'   # Prepare graph
+#'   nodes <- data.frame(name = c("Dom", "Mattan", "Daniel", "Brenton"))
+#'   edges <- data.frame(
+#'     from = c(1, 1, 1, 2, 3, 3, 4, 4, 4),
+#'     to = c(2, 3, 4, 1, 1, 2, 1, 2, 3)
+#'   )
+#'   data <- tidygraph::tbl_graph(nodes = nodes, edges = edges)
 #'
+#'   library(ggraph)
```
Comment (strengejacke) [inline R/geom_from_list.R:124]: I think we need to include this via `require()` as well.

File: tests/testthat/test-plot.cluster_analysis.R [hunk 3]
```diff
@@ -1,5 +1,6 @@
 test_that("`plot.see_cluster_analysis()` works", {
   library(parameters)
+  library(see)
```
Comment (strengejacke) [inline tests/testthat/test-plot.cluster_analysis.R:3]: packages should be included with `if-require()` or `if-requireNamespace()` - why do we need _see_ here?

File: tests/testthat/test-plot.easycormatrix.R [hunk 4]
```diff
@@ -1,5 +1,7 @@
 test_that("`plot.see_easycormatrix()` works", {
   library(correlation)
+  library(see)
+
```
Comment (strengejacke) [inline tests/testthat/test-plot.easycormatrix.R:4]: packages should be included with if-require() or if-requireNamespace() - why do we need see here?

File: tests/testthat/test-vdiffr_geoms_coords.R [hunk 5]
```diff
@@ -1,6 +1,4 @@
-.runThisTest <- Sys.getenv("RunAllseeTests") == "yes"
-
-if (.runThisTest && getRversion() >= "4.1" && require("vdiffr") && require("ggplot2")) {
+if (getRversion() >= "4.1" && getRversion() < "4.2" && require("vdiffr") && require("ggplot2")) {
   test_that("geom and coord functions work correctly", {
     # skip_on_cran()
```
Comment (strengejacke) [inline tests/testthat/test-vdiffr_geoms_coords.R:3]: should be skipped on CRAN.

**Review by strengejacke — CHANGES_REQUESTED** [review]

**Pull request merged**